controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -151,16 +155,19 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 			}
 		}
 
+		// TODO: Phases do not have checks at the moment, in order to make them more predictable and less error-prone, at the expense of increased computation cost.
```

Comment:
Ohad suggested that we remove the phase checks at the moment, for the aforementioned reason.